src/panels/ActionsPanel/ActionsPanel.test.js

```diff
@@ -60,6 +61,37 @@ describe("ActionsPanel", () => {
     expect(wrapper.find("Button").prop("disabled")).toBe(true);
   });
 
+  it("updates the title & unit list based on the number of units selected", async () => {
+    const wrapper = await generateComponent();
+    expect(wrapper.find("PanelHeader").text()).toBe(" 2 units selected");
+    expect(wrapper.find(".actions-panel__unit-list").text()).toBe(
```

Comment:
Select with a data-test in case this class changes?

Reply:
good call, done.